app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Api
+  module V1
+    class TargetsController < Api::V1::ApiController
+      def index
+        @targets = Target.all
+        render json: @targets
+      end
+
+      def create
+        @target = current_user.targets.create!(target_params)
```

Comment:
```suggestion
        @target = current_user.targets.build(target_params)
```
Because the `create!` will save the object into DB, so in this case, we want to 'build' it, and then check if we could save ir or not ;)